recipes/default.rb

```diff
@@ -43,7 +43,8 @@
   'rate-limiting'         => node['repose']['rate_limiting'        ]['cluster_id'],
   'slf4j-http-logging'    => node['repose']['slf4j_http_logging'   ]['cluster_id'],
   'translation'           => node['repose']['translation'          ]['cluster_id'],
-  'uri-identity'          => node['repose']['uri_identity'         ]['cluster_id']
+  'uri-identity'          => node['repose']['uri_identity'         ]['cluster_id'],
+  'api-validator'         => node['repose']['api_validator'        ]['cluster_id']
```

Comment:
I don't mind making it alph-beta.  I was attempting to defer to the 'standard filter order' suggested by the wiki, which has some loose recommendations:  https://repose.atlassian.net/wiki/display/REPOSE/Filters+and+services .   But if that's not an issue, then I'll adjust it.